src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,32 @@
+package calculator;
+
+public class Calculator {
+    private static final String DELIMITER = " ";
+    private static final Integer ZERO = 0;
+    private static final Integer INDEX_INTERVAL = 2;
+
+    private Calculator() {
+    }
+
+    public static int calculate(String input) {
+        inputValidate(input);
+
+        String[] arr = input.split(DELIMITER);
+        int sum = Integer.parseInt(arr[ZERO]);
+
+        for (int i = 1; i < arr.length; i += INDEX_INTERVAL) {
+            sum = calculator(sum , arr[i], Integer.parseInt(arr[i + 1]));
+        }
+        return sum;
+    }
+
+    private static int calculator(int firstOperand, String operator, int secondOperand) {
+        return Operator.of(operator).operate(firstOperand, secondOperand);
+    }
+
+    private static void inputValidate(String input) {
+        if (input == null || input == "") {
```

Comment:
```suggestion
        if (Objects.isNull(input) || input == EMPTY_STRING) {
```
해당 코드에서도 매직넘버를 사용하기 보단 의미를 가지는 상수로 정의하여 사용하는것이 좋아보입니다. 😄 
개인적으로 Objects.isNull을 사용하면 가독성을 더 높일수 있는것 같아 남겨드려요.